floris/wind_data.py

```diff
@@ -901,6 +985,87 @@ def plot_ti_over_ws(
         ax.set_ylabel("Mean Turbulence Intensity (%)")
         ax.grid(True)
 
+    @staticmethod
+    def read_csv_long(file_path: str,
+                       ws_col: str = 'wind_speeds',
+                       wd_col: str = 'wind_directions',
+                       ti_col: str = 'turbulence_intensities',
+                       freq_col: str | None = None,
+                       sep: str = ",",
+                       ) -> WindTIRose:
+        """
+        Read a long-formatted CSV file into the wind rose object. By long, what is meant
+        is that the wind speed, wind direction combination is given for each row in the
+        CSV file. The wind speed, wind direction, are
+        given in separate columns, and the frequency of occurrence of each combination
+        is given in a separate column. The frequency column is optional, and if not
+        provided, uniform frequency of all bins is assumed.
+
+        The value of ti_col_or_value can be either a string or a float. If it is a string,
+        it is assumed to be the name of the column in the CSV file that contains the
+        turbulence intensity values. If it is a float, it is assumed to be a constant
+        turbulence intensity value for all wind speed and direction combinations.
```

Comment:
The above summary part of the doctoring should be updated for the WindTIRose version

Reply:
Fixed, thank you